modules/ROOT/pages/use-tags-in-visualizer.adoc

```diff
@@ -9,7 +9,10 @@ If a tag is assigned to an application, the application will appear in every vie
 
 == Setting the Tag Through Properties
 
-To automate adding tag(s) to a node, use the property `anypoint.platform.visualizer.tags=<name>, <name>` in the application's properties file, or at deployment time through the Runtime Manager UI or API.
+To automate adding tag(s) to a node, use the property `anypoint.platform.visualizer.tags=<name>, <name>` in the application's properties file, or at deployment time through the Runtime Manager UI or API. 
+
+[NOTE]
+You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments.
```

Comment:
ditto previous edit questions and suggestions
Note also that if you keep "you can only use," switch it to "you can use only."

Reply:
That might be a touch confusing. You can set the property in CloudHub and RTF in ways besides the Runtime Manager UI. Here I'm stating Runtime Manager UI can only be used with those deployment types, not that those deployment types require that method. Does that make sense?

Reply:
```suggestion
You can automate adding tags to a node by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API. 

[NOTE]
You can only use the Runtime Manager UI for Cloudhub and Runtime Fabric deployments.
```